src/frontend/src/pages/ChangeRequestDetailPage/ActivationDetails.tsx

```diff
@@ -17,25 +17,33 @@ const ActivationDetails: React.FC<ActivationDetailsProps> = ({ cr }) => {
     <PageBlock title={'Activation Change Request Details'}>
       <Grid container>
         <Grid item xs={6} md={2}>
-          <b>Project Lead </b>
+          <Typography>
+            <b>Project Lead </b>
+          </Typography>
         </Grid>
         <Grid item xs={6} md={2}>
           {fullNamePipe(cr.projectLead)}
         </Grid>
         <Grid item xs={6} md={2}>
-          <b>Start Date </b>
+          <Typography>
+            <b>Start Date </b>
+          </Typography>
         </Grid>
         <Grid item xs={6} md={6}>
           {datePipe(cr.startDate)}
         </Grid>
         <Grid item xs={6} md={2}>
-          <b>Project Manager </b>
+          <Typography>
+            <b>Project Manager </b>
+          </Typography>
         </Grid>
         <Grid item xs={6} md={2}>
           {fullNamePipe(cr.projectManager)}
```

Comment:
pretty sure these pipes output strings and thus also need to be wrapped in Typography